uvicorn/workers.py

```diff
@@ -23,6 +24,12 @@ def run(self):
             "timeout_notify": self.timeout,
             "callback_notify": self.callback_notify,
         }
+
+        if self.cfg.is_ssl:
+            _ssl_opt = copy.deepcopy(self.cfg.ssl_options)
```

Comment:
I'd probably rather we explicitly copy across the options we need, rather than using `deepcopy`.

Reply:
gonna fix this

Reply:
I just made it a dictionary. Just for reference this is how `self.cfg.ssl_options` looks:
```python
{
    "keyfile": "./key.pem",
    "certfile": "./cert.pem",
    "ssl_version": 2,
    "cert_reqs": 0,
    "ca_certs": None,
    "suppress_ragged_eofs": True,
    "do_handshake_on_connect": False,
    "ciphers": "TLSv1",
}
```